README.md

```diff
@@ -31,7 +31,17 @@ Please remember to provide a good summary, description as well as steps to repro
 | [KeycloakClient](./deploy/crds/keycloak.org_keycloakclients_crd.yaml) | Represents a client in a keycloak server                 |
 | [KeycloakBackup](./deploy/crds/keycloak.org_keycloakbackups_crd.yaml) | Manage Keycloak database backups                         |
 
-## Deploying to a Cluster
+
+## Deployment to a Kubernetes or Openshift cluster
+
+The official documentation contains installation instruction for this Operator.
+
+[Getting started with keycloak-operator](https://www.keycloak.org/getting-started/getting-started-operator-kubernetes)
+
+[Keycloak Server Installation](https://www.keycloak.org/docs/latest/server_installation/index.html#_installing-operator)
```

Comment:
Probably should be "Operator installation"?

Reply:
Sure.